ui/src/plugins/com.android.AndroidLog/logs_panel.ts

```diff
@@ -381,6 +383,15 @@ export class LogsFilters implements m.ClassComponent<LogsFiltersAttrs> {
           });
         },
       }),
+      m(Button, {
```

Comment:
Nit: A `title` here saying 'Use regular expression' would be absolutely ideal, but I'm not going to block on this.